### DISCUSSION TITLE
BLE Audio

### DISCUSSION THREAD

**Discussion by JWZ1996** [Arkq/bluez-alsa#660] [General]
Is there a plan to add support for BLE audio and other BT 5.3 features like auracast?

I see that the newsy bluez and pipewire already have it.

I could help with that.

**Comment by arkq** [comment]
Yes, I've got a plan for that but before introducing BLE audio I will have to make little refactoring. I don't want to follow the BlueZ and PipeWire design where BLE audio is intermixed with A2DP. Anyway some things can be shared, so I will need to factor them out. But the timeline is rather unspecified... Now, I'm working on MIDI support which also requires some changes in BlueALSA service internals (I don't want to end up with a spaghetti code at the end)...

> I could help with that.

If you are willing to help that's great. I'll try to write down all my thoughts regarding the refactoring bit, so maybe it could be split into some small tasks.

**Reply by JWZ1996** [reply]
Hey, what about now? I feel I am ready to support you. How can we contact? I see that we are together from Poland, that's so convenient;)

I saw contribution readme, but this topic is very big so I wanted to be sure that we are on the same page

**Comment by chinli** [comment]
Hey, Is there any progress on this matter? Count me in please!
